.env.ci

```diff
@@ -142,3 +142,4 @@ E2E_TEST_ADMIN_EMAIL=dev+e2etest-admin@publiq.be
 E2E_TEST_ADMIN_PASSWORD=
 
 UITPAS_INTEGRATION_TYPE_ENABLED=true
+VITE_UITPAS_INTEGRATION_TYPE_ENABLED=true
```

Comment:
Would maybe make this `VITE_UITPAS_INTEGRATION_TYPE_ENABLED=${UITPAS_INTEGRATION_TYPE_ENABLED}` to ensure both stay in sync